### ISSUE TITLE
Why not make T4MVC_System_Web_Mvc_ActionResult as public class?

### ISSUE THREAD

**Issue by Valdis1987** [T4MVC/T4MVC#70]
Why not make T4MVC_System_Web_Mvc_ActionResult as public class?
It that case my code would be more testable and could verify class public properties.

**Comment by davidebbo** [comment]
Well, it feels a bit dirty to have all those generated classes be public, especially since they are generated in the global namespace (i.e. not in a namespace). In fact, there is a [test](https://github.com/T4MVC/T4MVC/blob/a7a090e6dc6fb095172b226e2db4dad1e116adf1/T4MVCHostMvcApp.Tests/T4MVCTest.cs#L819-L836) which specifically checks for that and would break.

**Comment by Valdis1987** [comment]
Yes of course it would get a litle bit dirty. But it would give to directly test code.
So there will be no changes for that class visibility?

**Comment by kevinkuszyk** [comment]
Have you considered using an `InternalsVisibleTo` attribute to make those classes accessible to your test code?
